### DISCUSSION TITLE
running srsUE on ADALM pluto board

### DISCUSSION THREAD

**Discussion by fpgasdr** [srsran/zynq_timestamping#26] [General] [answered]
Hi,

after been able to run the txrx application note with pluto board. i want to test the srsUE like with AntSDR but I got this error when running srsUE on ADALM pluto board 

ADALM pluto Error refilling buf -9

and not sure if this is due the bottleneck of bandwidth in usb2 interface or there other inherent problem I'm not aware of.

regards.

**Comment by ofontbach** [comment, marked as answer]
Hi @fpgasdr,

Thanks for spotting and reporting this. We actually noticed that an outdated file was unintentionally pushed for the RAMB primitive wrapping, which unfortunately meant that buffering was not well implemented in Zynq-7000 devices (which is the case for the Pluto board). We have just pushed the correct file (and updated the release to avoid future issues), and would like to kindly ask you to test this corrected version and let us know how it goes for you.

USB2 interface can be a bottleneck under larger real-time continuous data exchange conditions, but the txrx is only generating quite small data traffic.

Regards
